app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+import os.path
+
+
+def move_file(command: str) -> None:
+    cmd, file, new_file = command.split()
+
+    if len(command.split()) != 3 or cmd != "mv":
```

Comment:
It this case we already have an error:
Check this code:
```
command = "cp"
cmd, file, new_file = command.split()
if len(command.split()) != 3 or cmd != "mv":
     raise ValueError("Error#1")
```
You meed to validate len(command.split()) != 3 before upacking:
```
command = "cp"
parts = command.split()
if len(parts) != 3:
    raise ValueError("Description")

cmd, file, new_file = parts
if cmd != "mv" # also add check that files are not the same:
     raise ValueError("Description")
```